libs/features/mas/web-components/src/variants/product.js

```diff
@@ -56,6 +56,7 @@ export class Product extends VariantLayout {
     if (!isMobile()) {
       this.adjustProductBodySlots();
     }
+    this.adjustTitleWidth();
```

Comment:
should this be done for all card variants?
@yesil , @npeltier

Reply:
Since this has performance impact, I would limit it to variants we know this is needed.